tuned_lens/scripts/ingredients.py

```diff
@@ -91,8 +92,8 @@ class Model:
     name: str
     """Name of model to use in the Huggingface Hub."""
 
-    int8: bool = field(action="store_true", default=False)
-    """Load the model weights in mixed int8 mode with `bitsandbytes`."""
+    precision: Literal["auto", "bfloat16", "float16", "float32", "int8"] = "auto"
```

Comment:
I don't think simple parsing currently supports Literals. Can we make this an enum.

Reply:
I guess it does now.

Reply:
I caused simple parsing to support them as of v0.1.1